en/on-prem-prerequisites-and-installation.md

```diff
@@ -0,0 +1,135 @@
+| --------:     | ----------------  | ----------------- | ---------------------------------------   |-----  |
+| appbuilder    | 1.0               | 2a05977e039b      |12 days ago                                |854MB  |
+
+5 - Run the container:
+
+```
+docker run --restart always -p 80:5000 -e "ConnectionStrings:AppBuilderConnection=Data Source=<your-sql-database-ip>,<your-sql-database-port>;Database=<your-sql-database-name>;User ID=<your-sql-database-user>;Password=<your-sql-database-password>;Connect Timeout=15;Encrypt=False;TrustServerCertificate=False;ApplicationIntent=ReadWrite;MultiSubnetFailover=False" -v <external-folder-for-logs>:/appbuilder/logs -v <external-folder-for-storage>:/appbuilder/storage --name appbuilder appbuilder:1.0
+```
+
+6 - Open your browser and type `http://localhost/`
+
+Note: or if you are using Docker Desktop, go to Containers/Apps, find your container and click `Open in browser` 
+
+
+<img class="responsive-img" style="box-shadow: 5px -4px 13px 1px grey" src="./images/docker-apps.png" />
+<p style="text-align:center;">Docker Containers/Apps</p>
+
+### Updates
+
+
+1 - Follow the first 4 steps of the first time installation  with the newly published zip file.
+
+2 - Verify that the new image has loaded correctly (the old image should be tagged as <none>)
+
+```
+docker images
+```
+
+
+| REPOSITORY        | TAG       | IMAGE ID          | CREATED       |SIZE   |
+| --------:         | --------- | ----------------- | ------------- |-----  |
+| appbuilder        | 1.0       | 27ff4c1079ac      | 43 hours ago  |932MB  |
+| <none>            | <none>    | 2a05977e039b      | 12 days ago   |854MB  |
+        
+3 - Stop the container
+
+```
+docker stop appbuilder
+```
+
+4 - Remove the container
+
+```
+docker rm appbuilder
+```
+
+5 - Run the container with the same command as the one used in step 5. of the first time installation.
+
+
+## Activation
+This section assumes that you already installed the On-Prem instance and it is now up and running.
+
+When the server is first started a prompt dialog will provide you with _Installation ID_ and a _Authentication key_ will be requested. Send this Installation ID to our [Sales department](https://www.infragistics.com/about-us/contact-us#sales) based on your region and we will provide you with _Authentication key_ to activate the server.
+
+
+<img class="responsive-img" style="box-shadow: 5px -4px 13px 1px grey" src="./images/activate-app-builder.png" />
+<p style="text-align:center;">Activate App Builder</p>
+
+> [!NOTE]
+> You will be warn through the UI thirty days before the expiration of your key.
+
+
+## Troubleshooting
+### Docker Desktop on Windows
+[Docker Desktop on Windows is not starting automatically without Login to Windows machine](https://github.com/docker/for-win/issues/6670) - Docker team doesn't  recommend Docker Desktop for _production_ workloads. You should use Docker on a Linux box, or Docker for Windows Server if you want Windows containers.
```

Comment:
[Docker Desktop on Windows doesn't start automatically without Login to Windows machine](https://github.com/docker/for-win/issues/6670) - Docker team doesn't  recommend Docker Desktop for _production_ workloads. You should use Docker on a Linux box or Docker for Windows Server if you want Windows containers.